cfme/tests/cli/test_appliance_console.py

```diff
@@ -1220,9 +1220,8 @@ def test_appliance_console_logfile():
     pass
 
 
-@pytest.mark.manual
 @pytest.mark.tier(2)
-def test_appliance_console_restore_db_network_negative():
+def test_appliance_console_restore_db_network_negative(temp_appliance_preconfig_modscope):
```

Comment:
Q: Why the `modscope` temp appliance?

Reply:
This is negative test case if the test somehow failed and changed appliance behavior then it will make an impact on next test case

Reply:
Yes but as the `temp_appliance` is used for only this test case, shouldn't it be `function` scoped?

Reply:
Could you please tell me what should be expected application fixture here?